### ISSUE TITLE
32bit build vs Auto-upgrade

### ISSUE THREAD

**Issue by PizzaProgram** [gravitl/netclient#461]
## Please apply auto-build for 32bit netclient32.exe too!

Why?

1. Because [theoretically](https://www.digitalocean.com/community/tutorials/how-to-build-go-executables-for-multiple-platforms-on-ubuntu-20-04#step-5-creating-a-script-to-automate-cross-compilation) it would be easy to set up inside a bash here to auto-build

2. It would not conflict with built in **auto-upgrade** .
    - Currently if auto-upgrade would happen, it would probably stop the windows service,  
    - installing it's own latest 64bit version,  
    - which can not start, because it's not for 32bit.  
    - So basically it would **kill itself**, while locking out us, remote helpers from the system for good.

4. Also non-go programmers, like us would not have to deal with this on our own.
_(I've just installed GO on my PC, tried `E:\netclient-develop\go build` and it has downloaded 500+ MB to my `C:\...!!!` drive into various hidden directories without asking me ahead. )_

I would like to donate if this problem gets resolved for good.
Thank you very much in forward!
